### PR TITLE
Adding Platforms Section

### PR DESCRIPTION
**Pull request by rowanmiller**

### CODE REVIEW

File: entity-framework/core/platforms/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+# Platforms
+
+> [!NOTE]
+> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../ef6/index.md).
+
+We want EF Core to be available anywhere you write .NET code, and we're still working towards that goal. The following table provides guidance for each platform where we want to enable EF Core.
+
+<table>
```
Comment (rowanmiller) [inline entity-framework/core/platforms/index.md]: Using html as the markdown table syntax doesn't work very well for tables with this much in each cell

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C13%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201703131836042594-366%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rowanmiller** [comment]
Staged at https://review.docs.microsoft.com/en-us/ef/core/platforms/index?branch=platforms

**Comment by divega** [comment]
@Rick-Anderson @tdykstra do not merge this. It is blocked waiting for some additional testing on UWP.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C14%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201704142015234868-366%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C14%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201704142120151665-366%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C14%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201704142122581766-366%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/platforms/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/core/platforms/index.md) | :white_check_mark:Succeeded | |
[entity-framework/toc.md](https://github.com/aspnet/EntityFramework.Docs/blob/platforms/entity-framework/toc.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C6%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708061110051510-366%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**